modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/StatusRenderer.js

```diff
@@ -15,12 +15,33 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+function getLabelType(label) {
+	switch (label) {
+		case 'approved':
+			return 'success';
+		case 'denied':
+			return 'danger';
+		case 'expired':
+			return 'warning';
+		case 'draft':
+		case 'pending':
+		case 'scheduled':
+			return 'info';
+		default:
+			return '';
+	}
+}
+
 function StatusRenderer(props) {
 	return props.value ? (
 		<span className="taglib-workflow-status">
 			<span className="workflow-status">
 				<strong
-					className={`label status workflow-status-data-renderer workflow-status-${props.value.label} ${props.value.label} workflow-value`}
+					className={`label label-${getLabelType(
+						props.value.label
+					)} text-uppercase workflow-status-${props.value.label} ${
+						props.value.label
+					} workflow-value`}
```

Comment:
For complex CSS class setting, you can use the `classNames` util. See example [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/comment/comment-taglib/src/main/resources/META-INF/resources/discussion/js/components/UserIcon.js#L26-L28). Besides being more readable, it allows us to conditionally set some attributes, so we can have something like 
```js
[`label-${labelType}`]: !!labelType
```
In the solution in this PR, if `value.label` is unset or unknown, we would be left with dangling `label-` CSS class.

Reply:
Understood. A

As I don't completely get classNames usage and I have the option to return the complete label-type class from the helper function and it is very clean, I will do it this way.

```
const getLabelType = label => {
		switch (label) {
			case 'approved':
				return 'label-success';
			case 'denied':
				return 'label-danger';
			case 'expired':
				return 'label-warning';
			case 'draft':
			case 'pending':
			case 'scheduled':
				return 'label-info';
			default:
				return '';
		}
	}
```

Please let me know if this way could cause any harm or be a bad option

Reply:
That's a nice change! I prefer it like this, half-built classes are harder to read and hard to find when we mass-replace  :+1: 

There is a dangling space character, but that's not an issue. We can push this as it is.

About `classNames`, if you ever feel adventurous, see good and short tutorial on https://github.com/JedWatson/classnames#usage

Reply:
thank you!